service/history/task/transfer_active_task_executor.go

```diff
@@ -2050,9 +2052,11 @@ func applyParentClosePolicy(
 				Domain: childDomainName,
 				WorkflowExecution: &types.WorkflowExecution{
 					WorkflowID: childStartedWorkflowID,
-					RunID:      childStartedRunID,
```

Comment:
I am wondering if not setting the runId could cause some issues since from client side anyone can access the WorkflowExecution, what do you think? 
Was it removed from here because now we will be using FirstExecutionRunID everywhere?

Reply:
we use FirstExecutionRunID only here, and it doesn't make sense to set run id if FirstExecutionRunID is set